ISISComputingGroup/EPICS-inst_servers
# Ticket876 configurable logging for blocks

This adds per block configuration for logging in the archiver.

To Test:
* Add a block to a configuration that includes logging configuration data via either the json or xml file (see block_server.rst for format)
* Confirm that the behaviour of the logging into the archive database is as expected from the block_server.rst text

Note:
An error will occur unless [PR3](https://github.com/ISISComputingGroup/EPICS/pull/3) has been merged.
